website/content/docs/release-notes/consul-api-gateway/v0_4_x.mdx

```diff
@@ -0,0 +1,72 @@
+---
+layout: docs
+page_title: 0.4.x
+description: >-
+  Consul API Gateway release notes for version 0.4.x
+---
+
+# Consul API Gateway 0.4.0
+
+## Release Highlights
+
+- **Support for Kubernetes Gateway API Version 0.5.0 and v1beta1 APIs:**
+  The `v0.5.0` release of the Kubernetes Gateway API is significant because it
+  marks the growth in maturity to a beta API version (v1beta1) release for some
+  of the key APIs:
+  - GatewayClass
+  - Gateway
+  - HTTPRoute
+
+  The other APIs (e.g. TCPRoute) are still at the `v1alpha2` stage.
+
+  Reaching `v1beta1` status has several benefits for users, including greater
+  stability and backward compatibility requirements. Existing fields and allowed
+  options can not be removed or renamed except in a new, major version of the
+  API. Once an API reaches `v1beta1` status, future versions must comply with
+  several backward compatibility requirements.
```

Comment:
```suggestion
   The following API resources have been upgraded to `v1beta1` in Kubernetes Gateway API v0.5.0:

    - `GatewayClass`
    - `Gateway`
    - `HTTPRoute`

    These are key API endpoints Consul API Gateway implements to enable external network clients to access applications and services running in a Consul datacenter. 

  When an API reaches `v1beta1`, it has tighter stricter and backward compatibility requirements. Changing or removing existing fields and configuration options from the API, for example, requires a major version. Future versions must also comply with several backward compatibility requirements.
  
  Other API endpoints, such as `TCPRoute`, are still in `v1alpha2`.
  
  Refer to the [Kubernetes Gateway API documentation](https://kubernetes.io/blog/2022/07/13/gateway-api-graduates-to-beta/) for additional information about the API.
```
Does that flow any better?

Reply:
Suggested edits introduce minor inaccuracies.